src/main/kotlin/no/nav/syfo/identhendelse/database/IdenthendelseQuery.kt

```diff
@@ -1,28 +1,147 @@
 package no.nav.syfo.identhendelse.database
 
 import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
 import no.nav.syfo.domain.PersonIdentNumber
-import no.nav.syfo.oppfolgingstilfelle.bit.database.domain.POppfolgingstilfelleBit
+import java.sql.Connection
+import java.sql.PreparedStatement
 
-const val queryUpdateTilfelleBitFnr =
+const val queryUpdateTilfelleBit =
     """
         UPDATE TILFELLE_BIT
         SET personident = ?, processed = FALSE
         WHERE personident = ?
     """
 
-fun DatabaseInterface.updateTilfelleBit(nyPersonident: PersonIdentNumber, tilfelleBitWithOldIdentList: List<POppfolgingstilfelleBit>): Int {
+fun Connection.updateTilfelleBit(
+    nyPersonident: PersonIdentNumber,
+    inactiveIdenter: List<PersonIdentNumber>,
+    commit: Boolean = false,
+): Int {
+    return this.updateIdent(
+        query = queryUpdateTilfelleBit,
+        nyPersonident = nyPersonident,
+        inactiveIdenter = inactiveIdenter,
+        commit = commit,
+    )
+}
+
+const val queryUpdateOppfolgingstilfellePerson =
+    """
+        UPDATE OPPFOLGINGSTILFELLE_PERSON
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun Connection.updateOppfolgingstilfellePerson(
+    nyPersonident: PersonIdentNumber,
+    inactiveIdenter: List<PersonIdentNumber>,
+    commit: Boolean = false,
+): Int {
+    return this.updateIdent(
+        query = queryUpdateOppfolgingstilfellePerson,
+        nyPersonident = nyPersonident,
+        inactiveIdenter = inactiveIdenter,
+        commit = commit,
+    )
+}
+
+const val queryUpdatePerson =
```

Comment:
Tror det blir best å ta bort dette fordi: 
1. Det er bare identer for døde personer som legges finnes her
2. Når vi får inn et dødsfall så legges dødsdato inn her for alle personens identer (dvs en hendelse kan generere flere rader i tabellen)
3. Det er en unique-constraint på personidenten, så dette kan generere en feil (siden det allerede finnes en rad for en ident man forsøker å endre til).